package.json

```diff
@@ -49,7 +51,7 @@
     "eslint-plugin-cypress": "^2.13.3",
     "eslint-plugin-import": "^2.27.5",
     "eslint-plugin-jest": "^27.2.1",
-    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-n": "^16.0.0",
```

Comment:
Is this meant to be of this pull request?

Reply:
I was getting intermittent deprecation errors using this package in ESLint. It turns out `eslint-plugin-n` is recommended.
I agree it's outside the scope of the PR, but for such a minor change, that'll save from having to disable the CI to merge the PR